{"name": "Принятие заказа с неверным ID заказа", "status": "passed", "start": 1727815815603, "stop": 1727815816217, "uuid": "9777cbde-f145-44bd-a90c-d0dde4bc62d3", "historyId": "e4f726252d51a7f0f084ca15b11fa1cc", "testCaseId": "e4f726252d51a7f0f084ca15b11fa1cc", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_invalid_order_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "POPOV"}, {"name": "thread", "value": "30656-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}